android/debugging_tips.md

```diff
@@ -0,0 +1,14 @@
+# Debugging Tips
+
+
+### Profiling code snippits
+
+You easily profile a section of your application programatically starting and stopping the profiler.
+
+```
+Debug.startMethodTracingSampling("${context.filesDir}/something.trace", 0, 100)
+// Code to profile
+Debug.stopMethodTracing()
+```
+
+You can then find the results in the *Device File Explorer* under the path `Data > Data > org.mozilla.app.debug > something.trace` and load it in the profiler
```

Comment:
nit: `Data` to lowercase.

Also, I think it's `org.mozilla.app.debug/files/something.trace`, right?

Also, wonder if we should just make it a path: `/data/data/org.mozilla.app.debug/...`